### ISSUE TITLE
SsdMobilenetv1 - load model before inference

### ISSUE THREAD

**Issue by ShadabFaiz** [justadudewhohacks/face-api.js#189]
This is how i'm loading the modules.
``` typescript
        await this.loadTinyModel();
        console.log('model loaded');
        let canvas = await this.loadCanvas(filePath);
        let tt = await faceapi.detectAllFaces(canvas);

    private async loadTinyModel() {
        await faceapi.nets.faceLandmark68Net.loadFromDisk(path.join(__dirname, 'models'));
    }

    private async loadCanvas(filePath: string) {
        return canvas.loadImage(filePath);
    }
```

But getting errors:
``` node
(node:27555) UnhandledPromiseRejectionWarning: Error: SsdMobilenetv1 - load model before inference
    at SsdMobilenetv1.forwardInput (/home/Faiz/Organization/Repos/Image_Analysis/using_FaceAPI/node_modules/face-api.js/src/ssdMobilenetv1/SsdMobilenetv1.ts:26:13)
    at SsdMobilenetv1.<anonymous> (/home/Faiz/Organization/Repos/Image_Analysis/using_FaceAPI/node_modules/face-api.js/src/ssdMobilenetv1/SsdMobilenetv1.ts:60:14)
    at step (/home/Faiz/Organization/Repos/Image_Analysis/using_FaceAPI/node_modules/tslib/tslib.js:133:27)
    at Object.next (/home/Faiz/Organization/Repos/Image_Analysis/using_FaceAPI/node_modules/tslib/tslib.js:114:57)
    at fulfilled (/home/Faiz/Organization/Repos/Image_Analysis/using_FaceAPI/node_modules/tslib/tslib.js:104:62)
    at <anonymous>
(node:27555) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:27555) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by justadudewhohacks** [comment]
You are calling detectAllFaces, which assumes (by default) that the ssdMobilenetv1 is loaded:
`await faceapi.nets.ssdMobilenetv1.loadFromDisk(path.join(__dirname, 'models'))`.

**Comment by ckahle33** [comment]
Also seeing this error trying to use `detectSingleFace()`;

```
import * as faceapi from 'face-api.js';

const load = async function() {
  const net = new faceapi.TinyFaceDetector
  await net.load('/assets')
  // await faceapi.loadTinyFaceDetectorModel('/assets')
}

const detect = async function() {
  const image = document.getElementById('detect')
  const detection = await faceapi.detectSingleFace(image)
  console.log(image)
}

document.addEventListener("DOMContentLoaded", function(event) {
  load()
  detect()
});
```

Throws this similar error: 
```
SsdMobilenetv1.js:20 Uncaught (in promise) Error: SsdMobilenetv1 - load model before inference
    at SsdMobilenetv1.forwardInput (SsdMobilenetv1.js:20)
    at SsdMobilenetv1.<anonymous> (SsdMobilenetv1.js:54)
    at step (tslib.es6.js:97)
    at Object.next (tslib.es6.js:78)
    at fulfilled (tslib.es6.js:68)
```
Thanks for your time in creating this library.

**Comment by rudevich** [comment]
So `SsdMobileNetv1` always should be loaded?
Could I use `tinyFaceDetector` only?

**Comment by Deva2896** [comment]
> So `SsdMobileNetv1` always should be loaded?
> Could I use `tinyFaceDetector` only?
> 
> UPDATE: sorry, found an answer https://github.com/justadudewhohacks/face-api.js#detecting-faces :)

what you add in youe old code?i have same problem brother.can you tell mi how you solve it.

**Comment by v3da-bit** [comment]
> So `SsdMobileNetv1` always should be loaded? Could I use `tinyFaceDetector` only?
> 
> UPDATE: sorry, found an answer https://github.com/justadudewhohacks/face-api.js#detecting-faces :)

which line of code have you added in your program to remove this error please tell me! I'm facing the same issue, and can't able to solve it

**Comment by gauravdubey19** [comment]
I encountered the same error in my Next.js project. The issue was due to importing `loadModule()` from the server side ( server actions ), which is responsible for loading the modules.

Initially, the modules load without any issues. However, when I try to execute the following code:

```
const detections = await faceapi.detectAllFaces(input).withFaceLandmarks();
console.log(detections);
```
I get the same error again. It appears that this functionality only works correctly on the client side.
